src/api/text-annotator.ts

```diff
@@ -0,0 +1,82 @@
+import { biolink, ApiError, request, cleanError } from "./index";
+
+interface Response {
+  content: string;
+  spans: Array<{
+    start: number;
+    end: number;
+    text: string;
+    token: Array<{
+      id: string;
+      category: Array<string>;
```

Comment:
Just FYI, this should change to `categories` if my proposed fix for biolink is accepted.

Reply:
Can you tag me in that PR so I know when it's merged